src/clients/firstboot_licenses.rb

```diff
@@ -71,11 +71,7 @@ def main
 
       @result = WFM.CallFunction("inst_license", [@args])
 
-      if @result == :halt
-        UI.CloseDialog
-        Builtins.y2milestone("Halting the system...")
-        SCR.Execute(path(".target.bash"), "/sbin/halt")
```

Comment:
Um, where is this `/sbin/halt` executed now?

Reply:
It is now handled by the Service, I have unified it and now instead of each client calling halt directly halt, they just return :halt and is the `firstboot client` [here](https://github.com/yast/yast-firstboot/pull/49/files#diff-1d6918861bf68a8e44766d2cc8cad4caR112) who touch a flag file which is then checked by the S09-cleanup script for returning exit code 0 in that case

Then the YaST2-Firsboot.Service calls shutdown on failure.